_docs/principles/accessibility.md

```diff
@@ -3,29 +3,69 @@ title: Accessibility
 layout: docs
 category: Principles
 usage: >
-  Visual accessibility standards enable users with varied levels of color impairment to successfully navigate and use a Helix-based UI.
+  Accessibility standards promote the design of user interfaces (UI) so they can be used by people with the widest range of abilities within the widest range of situations.
 status: stable
 resource: true
-last-modified: 2017-10-24
+last-modified: 2019-05-06
 ---
 
 {% include toc.html %}
 
 <section class="static-section" markdown="1">
 
-## Accessibility for visual design
-
 <div class="hxRow"  markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Understanding accessibility
+
+An accessible UI is designed so that it can be *accessed* and used by people who experience a permanent or a temporary impairment. An impairment can be visual, auditory, physical, or cognitive.
+
+For example, a user who suffers from a concussion may experience difficulty processing information, so ensure that you write succinct on-page instructions and labels. A user who experiences hand tremors as a side effect to medication has difficulty operating a mouse, so design pointer targets so they are large enough.
 
-Rackspace products must be made accessible for everyone&mdash;regardless of a person’s abilities. We can assume from the statistics that many of our users have some form of visual impairment. [Colour Blind Awareness](http://www.colourblindawareness.org/) estimates that:
+Designing for accessibility is a best practice when using Helix because your designs will be usable by everyone regardless of whether they have any impairments.
+
+The purpose of this document is to articulate guidelines that you can use when you design control panel page. The list of guidelines is not exhaustive. Refer to [Web Content Accessibility Guidelines 2.1 (WCAG)](https://www.w3.org/TR/WCAG21/) for a complete list of guidelines.
+
+### Why accessibility is important
+
+Accessibility is more than just good design. It's about the empathy for and inclusion of all people. Consider sight. [Colour Blind Awareness](http://www.colourblindawareness.org/) estimates that:
```

Comment:
never mind... this is a proper noun

Reply:
I didn't change it when I edited it because it is what is used on the site... I suggested adding 
"site" after Awareness because it is a bit jarring. Adding site seemed to help, especially if the link is followed and users see the British spelling there.